src/main/reactjs/src/styles/styles.scss

```diff
@@ -16,10 +16,10 @@
 // Components
 @import 'components/clerkTranslator/authorisation-details';
 @import 'components/clerkTranslator/clerk-translator-autocomplete-filters';
-@import 'components/elements/custom-circular-progress';
```

Comment:
tämä on vielä käytössä. Oliko joku syy poistamiseen?

Reply:
Lisäsin tänne aiemmin importin `components/elements/date-picker` jonka myöhemmin poistin. Tää import on edelleen käytössä, mutta se on vaan uudelleenlisättynä tuossa alempana. En revertannut näitä muutoksia, sillä asettivat nämä aakkosjärjestykseen toisiinsa nähden.